packages/client/hmi-client/src/components/workflow/ops/simulate-ciemss/tera-simulate-node-ciemss.vue

```diff
@@ -126,15 +128,23 @@ Provide a summary in 100 words or less.
 
 	const summaryResponse = await createLLMSummary(prompt);
 
+	const datasetName = `Forecast run ${runId}`;
+	const projectId = useProjects().activeProject.value?.id ?? '';
+	const datasetResult = await createTemporaryDatasetFromSimulationResult(projectId, runId, datasetName);
+	if (!datasetResult) {
+		logger.error('Error creating dataset from simulation result.');
+		return;
+	}
```

Comment:
We have something called `createDatasetFromSimulationResult` - I am not familiar with it, but have we looked to see if we can use it or extend the feature to adapt to this use case? Cause at the surface it seems like you are doing very similar things.

https://github.com/DARPA-ASKEM/terarium/blob/412a24f7b5fc14d98dde66d72e21d3de111d6bde/packages/client/hmi-client/src/services/dataset.ts#L273-L276

Reply:
I can add a flag to `createDatasetFromSimulationResult` for temporary 
And change the return type from boolean -> Dataset
as well as update the use of it in `saveDataset` to still have that boolean logic there

Good call give me a moment